src/DataWrangling/ETOPO/ETOPO.jl

```diff
@@ -0,0 +1,84 @@
+module ETOPO
+
+using Scratch
+export ETOPOBathymetry
+
+import ClimaOcean.DataWrangling:
+    metadata_filename,
+    default_download_directory,
+    metadata_path,
+    Metadata,
+    Metadatum,
+    all_dates,
+    first_date,
+    last_date,
+    download_dataset,
+    download_progress,
+    longitude_interfaces,
+    latitude_interfaces,
+    is_three_dimensional,
+    z_interfaces,
+    reversed_vertical_axis
+
+using Downloads
+
+using Oceananigans
+using Oceananigans.DistributedComputations: @root
+
+download_ETOPO_cache::String = ""
+function __init__()
+    global download_ETOPO_cache = @get_scratch!("ETOPO")
+end
+
+ETOPO_bathymetry_variable_names = Dict(
+    :temperature => "temperature",
+    :salinity    => "salinity"
+)
```

Comment:
```suggestion
```
I guess this is vestigial code remaining from copying other datasets

Reply:
Yeah, we probably don't need it. At the moment it's not used.

Reply:
well I think it is also wrong, the dataset only has a bottom height